src/soc/intel/baytrail/pcie.c

```diff
@@ -81,10 +79,12 @@ static void byt_pcie_init(struct device *dev)
 		REG_PCI_RMW32(LCAP, ~L1EXIT_MASK,
 			2 << (L1EXIT_SHIFT + pll_en_off)),
 		REG_SCRIPT_NEXT(init_static_after_exit_latency),
+#if !CONFIG(PCIEXP_HOTPLUG)
 		/* Disable hot plug, set power to 10W, set slot number. */
 		REG_PCI_RMW32(SLCAP, ~(HPC | HPS),
 			(1 << SLS_SHIFT) | (100 << SLV_SHIFT) |
 			(root_port_offset(dev) << SLN_SHIFT)),
```

Comment:
I may have misunderstood BWG, because I though it gives me the order of operations to be done. SO if downstream device is not detected, the new should skip the slot power limit setting?

Reply:
@krystian-hebel  IIUC as of [this commit](https://github.com/Dasharo/coreboot/pull/542/commits/bff6f1a2e42185d2a2f029412b6b56ed71849f04) hotplug is disabled if specified in the config file, whereas power and slot number get set regardless - not sure if I got it right it though